modules/ROOT/pages/find-and-fix-conformance-issues.adoc

```diff
@@ -1,30 +1,22 @@
 = Finding and Fixing Conformance Issues
 :page-aliases: view-conformance-status-in-exchange.adoc
 
-To find and fix API conformance:
+As you govern and monitor APIs for conformance, you can continually improve conformance by empowering developers and implementors to find and fix conformance issues.
 
-. Initiate validation of APIs against rulesets in:
-+
-* xref:create-profiles.adoc[API Governance] for all supported API aspects
-* xref:find-conformance-issues.adoc[Design Center] for  specifications only
-* xref:find-conformance-issues.adoc#validate-in-cli[Anypoint CLI] for specifications only
-+
-. View conformance information as follows: 
-+
-* For an API validated using active profiles in API Governance, view conformance information in the:
-+
-** <<governance-validation-report,Validation report in API Governance>> for all supported API aspects
-** <<view-conformance-in-exchange,Conformance Status page for APIs in Exchange>> for specifications only
-** <<view-governed-apis-for-rulesets,Governed APIs page for rulesets in Exchange>> for specifications only
-+ 
-* To view conformance information for an API regardless of whether it's governed by an active profile, you can use:
-+
-** xref:find-conformance-issues.adoc#project-errors[Design Center] for specifications only
-** xref:find-conformance-issues.adoc#validate-in-cli[Reports generated through CLI] for specifications only
-+
-. Fix the issues using:
-+
-** xref:find-conformance-issues.adoc#project-errors[Design Center] for specifications
+Governance admins typically:
+
+* Initiate validation of APIs against rulesets in xref:create-profiles.adoc[API Governance] for all supported API aspects
+* View conformance information in the <<governance-validation-report,validation report in API Governance>>
+* Notify developers or implementors about conformance issues
+
+Developers and implementors typically:
+
+* Initiate validation of API specifications against rulesets in xref:find-conformance-issues.adoc[Anypoint Code Builder or Design Center].
+* View API specification conformance information in the <<view-conformance-in-exchange,Conformance Status page for specifications>> or in the
```

Comment:
When you validate against the spec from the IDE, you also get a report in the IDE indicating what passes and fails.

Reply:
Good point. ACB is a more rich set of info than APID, too. I'll reword to avoid giving that impression.